### ISSUE TITLE
Doubt Regarding Normalization

### ISSUE THREAD

**Issue by sheoranhimansh** [xingyizhou/pytorch-pose-hg-3d#66]
Hi,
Before training the Model are you doing any kind of normalization of the joints? Like shifting the origin to the coordinates of pelvis or head, or subtracting the mean from the joint location and dividing by its standard deviation.

Because the thing is that while calculating MSE loss on the original joint location - square((pred_x-gt_x)) the loss is very large and subsequently takes more time to converge. I noticed that my loss is of order of 10^2 whereas in your log file the loss if of magnitude 10^-3. 

What kind of changes did you make in your groundtruth_x before training the model?

Thank You
